config/development.config.php.dist

```diff
@@ -24,8 +24,12 @@ declare(strict_types=1);
  */
 
 use Laminas\ConfigAggregator\ConfigAggregator;
+use Mezzio\Tooling\ConfigProvider as ToolingConfigProvider;
 
 return [
-    'debug'                        => true,
-    ConfigAggregator::ENABLE_CACHE => false,
+    ...[
+        'debug'                        => true,
+        ConfigAggregator::ENABLE_CACHE => false,
+    ],
```

Comment:
The above allows mezzio-tooling to be enabled in development, without requiring it in `config/config.php`.